tools/disaster-recovery/d1-restore-drill.ts

```
@@ -0,0 +1,874 @@
+
+export function verifyRows(
+	query: VerificationQuery,
+	rows: Array<QueryRow>,
+	baseline: Readonly<RestoreBaseline>,
+): void {
+	switch (query.id) {
+		case 'integrity':
+			if (rows.length !== 1) {
+				throw new Error(`${query.phase} PRAGMA quick_check failed`)
+			}
+			const quickCheckEntries = Object.entries(rows[0] ?? {})
```

Biome (2.5.3)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)